### DISCUSSION TITLE
can't uninstall jellyfin properly

### DISCUSSION THREAD

**Discussion by techno-disaster** [jellyfin/jellyfin#6885] [Q&A]
I clearly should not have done it, but I decided to nuke /var/lib/jellyfin because some of my metadata was wrong and I couldn't figure out where it was. But then jellyfin did not work so I purged it and reinstalled but not I can't create a new user because it just sends me back to the old login page (yes the old one, with my old theme, this should be deleted  but idk where it's cached) I tried clearing browser cache too. I also could not sign in with my old credentials. Can someone help me out here?

**Comment by techno-disaster** [comment]
After some more tinkering I'm now able to login and will setup everything from scratch, can someone still confirm where does jellyfin store metadata for shows? I changed the title of one episode using the "Edit metadata" option but then could not go back to the original filename even after rescanning and readding the library

**Comment by aransou** [comment]
I think this will be stored in the internal sqllite database. You can edit it there or you renew your system/scan.
